docs/managed-victoriametrics/alertmanager-setup-for-deployment.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Alertmanager and VMAlert configuration for Deployment
+aliases:
+- /managed-victoriametrics/alertmanager-configuration.html
+---
+
+## Alerting stack configuration and Managed VictoriaMetrics
+
+ Managed VictoriaMetrics supports configuration for alerting rules and notifications for it with alertmanager.
+
+## Configure alertmanager
+
+ Managed VictoriaMetrics supports alertmanager with standard [configuration](https://prometheus.io/docs/alerting/latest/configuration/).
```

Comment:
Managed VictoriaMetrics supports Alertmanager with a standard [configuration](https://prometheus.io/docs/alerting/latest/configuration/). The configuration menu is located on the deployment page under the Alertmanager section.